packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -429,6 +429,18 @@ export const schema: SchemaType<DbTag> = {
       label: val.label
     })),
   },
+
+  canVoteOnRels: {
+    type: Array,
+    canRead: ['guests'],
+    canUpdate: ['admins', 'sunshineRegiment'],
+    canCreate: ['admins', 'sunshineRegiment'],
+    optional: true,
+    hidden: true,
```

Comment:
Why not make this editable by admins? Seems like that's the main benefit to having this field on the tag collection

Reply:
Good call! I've made the change!